hcl2template/types.build.go

```diff
@@ -60,6 +60,9 @@ func (p *Parser) decodeBuildConfig(block *hcl.Block) (*BuildBlock, hcl.Diagnosti
 		Config      hcl.Body `hcl:",remain"`
 	}
 	diags := gohcl.DecodeBody(block.Body, nil, &b)
+	if diags.HasErrors() {
```

Comment:
nice!! 👍  If there's an error we should fail faster :D

Reply:
yes !